src/drivers/riscv_cpu.c

```diff
@@ -141,7 +141,7 @@ void __metal_exception_handler (void) {
     		uintptr_t mtvt;
     		metal_interrupt_handler_t mtvt_handler;
 
-                asm volatile ("csrr %0, mtvt" : "=r"(mtvt));
+                asm volatile ("csrr %0, 0x307" : "=r"(mtvt));
```

Comment:
Is this to get around RISC-V toolchains without CLIC support?

We should probably `#define` this CSR number somewhere.

Reply:
Does not make sense to add a metal-car.s file for just one register now. Will add one if more is needed.